server/utils/utils.ts

```diff
@@ -21,3 +23,30 @@ export const initialiseName = (fullName?: string): string | null => {
   const array = fullName.split(' ')
   return `${array[0][0]}. ${array.reverse()[0]}`
 }
+
+export const switchDateFormat = (displayDate: string, fromFormat = 'dd/MM/yyyy') => {
+  if (displayDate) {
+    return DateTime.fromFormat(displayDate, fromFormat, { locale: 'en-GB' }).toISODate()
+  }
+  return displayDate
+}
+
+export const getCurrentPeriod = (dt: DateTime): string => {
+  const afternoonSplit = 12
+  const eveningSplit = 17
+  if (dt.hour < afternoonSplit) return 'AM'
+  if (dt.hour < eveningSplit) return 'PM'
+  return 'ED'
+}
+
+export const arrayToQueryString = (array: string[] | number[] | boolean[], key: string): string =>
+  array && array.map(item => `${key}=${encodeURIComponent(item)}`).join('&')
```

Comment:
Maybe use URLSearchParams? https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/toString

```suggestion
export const arrayToQueryString = (array: string[] | number[] | boolean[], key: string): string => new UrlSearchParams(array).toString()
```

Reply:
Will this work? cool. I had just lifted this from dps

Reply:
Not sure, test it first 😄

Reply:
Wasnt needed for this PR - deleted - will fallow your suggestion if i come across it again